content/docs/command-reference/plots/modify.md

```diff
@@ -134,8 +134,8 @@ plots:
 
 ## Example: Template change
 
-_dvc stage add --plots file.csv ..._ command assign the default template that
-needs to be changed in many cases. A simple command changes the template:
+Something like `dvc stage add --plots file.csv ...` assigns the default
```

Comment:
We can auto-link to https://dvc.org/doc/command-reference/stage/add#--plots directly from here now.